RadioItem/RadioItem.module.less

```diff
@@ -18,10 +14,28 @@
 			border-radius: @agate-radioitem-icon-border-radius;
 			color: transparent;
 			opacity: @agate-radioitem-icon-opacity;
+			margin: @agate-radioitem-icon-margin;
+			display: flex;
+			align-items: center;
+			justify-content: center;
+		}
+
+		&.inline {
+			&.item {
+				padding: @agate-radioitem-inline-padding;
+			}
+		}
+
+		.slotBefore {
+			padding: @agate-radioitem-slotbefore-padding;
```

Comment:
Do we have a guid for this?
![image](https://user-images.githubusercontent.com/5037429/118225072-543d7880-b4bf-11eb-92aa-bce979e16b6d.png)

Reply:
I added padding here to follow this requirement:
![padding](https://user-images.githubusercontent.com/63335823/118238880-cbfcaa80-b4a1-11eb-9bfe-aa9a360cd59c.png)

..but since it breaks layout when `slotBefore` icon is used, I will remove it.
I will add margin to icon to follow the above requirement, and this way the `slotBefore` won't be affected